### ISSUE TITLE
Failing test with Python 3.8

### ISSUE THREAD

**Issue by infertux** [Electron-Cash/Electron-Cash#1923]
```
$ tox -e py38
[...]
lib/tests/test_interface.py::TestInterface::test_check_host_name FAILED
[...]
ValueError: too many values to unpack (expected 2)
```

BTW I noticed there [doesn't seem to be any Travis CI setup](https://travis-ci.org/Electron-Cash/Electron-Cash) even though a [.travis.yml](https://github.com/Electron-Cash/Electron-Cash/blob/master/.travis.yml) file is present. Is there another CI setup I didn't see?

**Comment by hegjon** [comment]
I'm getting the same error on Fedora 32 running `$ tox -e py38`. Git commit e31d53c334e0a3a444062cbfbe1d14d4ecac4322

**Comment by EchterAgo** [comment]
I fixed the test in #1929 and made our Travis CI setup work again in #1930. After this is merged @fyookball would have to enable https://travis-ci.org/ as well as https://coveralls.io/ on the repository.

**Comment by infertux** [comment]
Awesome! Thank you @EchterAgo :)
